samples/boards/nrf/system_off/prj.conf

```diff
@@ -1,5 +1,4 @@
 # Required to disable default behavior of deep sleep on timeout
-CONFIG_PM_DEVICE=y
```

Comment:
This should stay. The purpose of this sample is not only to demonstrate System OFF mode:
https://github.com/zephyrproject-rtos/zephyr/blob/0c55710040747d5984a2594a3061c988519d5ac3/samples/boards/nrf/system_off/README.rst#L9-L18
Moreover, the UART needs to be suspended before entering System OFF (the sample lacks that currently), otherwise the power consumption is above 200 uA instead of a few uA (before 37b26f40a6f671499c192a1ac30e32b633b3378f, the UART was suspended by `pm_state_force`, now it needs to be done explicitly).

Reply:
so we need to find an alternative to the `pm_device_action_run`, it is wrong to use that API directly.

Reply:
this is actually yet another case of misunderstanding what CONFIG_PM_DEVICE is about, ref https://github.com/zephyrproject-rtos/zephyr/pull/62160

Reply:
@anangl would it be sufficient to add `disable-rx` to the UART in an overlay?

Reply:
> @anangl would it be sufficient to add `disable-rx` to the UART in an overlay?

I'll check.

Reply:
I've checked this. Unfortunately, disabling RX makes difference only in the "Sleep 2 s" state (the current consumption is noticeably lower), but not in System OFF. It must be the enabled UART, not just its receiver, that causes these ~217 uA to be drawn.

Reply:
Ok, I'll leave the pm_device_action_run and try to find an alternative in the near future.